README.md

```diff
@@ -100,30 +58,25 @@ yarn test
 
 Steps to install and set up:
 
+```
+conda env create -f environment_dev.yml
+conda activate cs-ui-dev-env
+playwright install chromium
+cp .env.example .env
+corepack enable
+yarn install --immutable
+yarn build
+```
+
+Line by line, here's what the commands above do:
+
```

Comment:
Is there a reason why splitting commands then explanations?

Reply:
I believe you can copy and paste the commands all at once